WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
 class SigninEditingState {
-    static var signinEditingStateActive = false
+    static var signinEditingStateActive = true
```

Comment:
Ok, I reverted this because the _proper_ fix got well beyond the scope of this PR. I've got it sitting in a local branch in case I get time to finish it.